src/support/z_utils.erl

```diff
@@ -430,11 +408,6 @@ js_prop_value(_, {trust, Value}, _OptContext) -> Value;
 js_prop_value(_, Value, OptContext) -> [$",js_escape(Value, OptContext),$"].
 
 
-%% @doc Deprecated: moved to z_json.
-json_escape(A) ->
-    z_json:json_escape(A).
-
-
 only_letters([]) ->
     true;
 only_letters([C|T]) when (C >= $a andalso C =< $z) orelse (C >= $A andalso C =< $Z) ->
```

Comment:
Line 413 is too long: only_letters([C|T]) when (C >= $a andalso C =< $z) orelse (C >= $A andalso C =< $Z) ->.